docs/start/install.mdx

```diff
@@ -360,6 +360,16 @@ release had some issue with the sdist package publishing, and a post-release
 0.25.2.1 only fixed the packaging issue for the release.
 </details>
 
+<details>
+<summary id="qpy-support">
+QPY support</summary>
+The QPY serialization format is backwards compatible so that a new Qiskit can release can always load a QPY
+file generated with an earlier release of Qiskit. However, the format isn't forward compatible so, in principle, it's not possible
+to load QPY files generated with newer version of Qiskit using an older release.  To facilitate user migration across major version releases the `qiskit.qpy.dump()` function will always support at least one overlapping version between the `X.0.0` and the `X-1.Y.0` release (where `Y` is the last minor version of 
+that series). This will enable saving QPY format files that can be loaded by both major versions from the newer
+release. See more details in [RFC 0020](https://github.com/Qiskit/RFCs/blob/master/0020-release_cycle.md#qpy-support).
```

Comment:
```suggestion
The QPY serialization format is backwards-compatible so that a new Qiskit release can always load a QPY
file generated with an earlier release of Qiskit. However, the format isn't forward-compatible so, in principle, it's not possible
to load QPY files generated with a newer version of Qiskit using an older release.  To facilitate user migration across major version releases, the `qiskit.qpy.dump()` function will always support at least one overlapping version between the `X.0.0` and the `X-1.Y.0` release (where `Y` is the last minor version of 
that series). This will enable saving QPY format files that can be loaded by both major versions from the newer
release. See more details in [RFC 0020](https://github.com/Qiskit/RFCs/blob/master/0020-release_cycle.md#qpy-support).
```